packages/utils/src/useTableTools/TableToolTable.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Pagination, PaginationVariant } from '@patternfly/react-core';
+import { Table, TableBody, TableHeader, sortable, fitContent } from '@patternfly/react-table';
+import PrimaryToolbar from '@redhat-cloud-services/frontend-components/PrimaryToolbar';
+import TableToolbar from '@redhat-cloud-services/frontend-components/TableToolbar';
+
+import useTableTools from './useTableTools';
+
+const severityLevels = [ 'high', 'medium', 'low', 'unknown' ];
+const exampleItems = (count) => {
+    new Array(count).map((count) => {
+        const randomNumber = Math.floor(Math.random() * severityLevels.length);
+        return {
+            severity: severityLevels[ randomNumber ],
+            name: `Test Item ${ count }`
+        };
+    });
+};
+
+/* eslint-disable react/display-name */
+export const exampleColumns = [
+    {
+        title: 'Name',
+        prop: 'name',
+        transforms: [ sortable ],
+        sortByProperty: 'name',
+        renderFunc: (name) => (
+            name
+        )
+    }, {
+        title: 'Severity sorted by function',
+        transforms: [ sortable ],
+        sortByFunction: (item) => (severityLevels.indexOf(item.severity)),
+        renderFunc: (item) => (
+            item.severity
+        )
+    }, {
+        title: 'Severity by Array',
+        transforms: [ sortable, fitContent ],
+        sortByArray: severityLevels,
+        renderFunc: (item) => (
+            item.severity
+        )
+    }
+];
+/* eslint-enable */
+
+const ExampleTable = ({ items = exampleItems(100), columns = exampleColumns }) => {
+    const { toolbarProps, tableProps } = useTableTools(items, columns);
+
+    return <React.Fragment>
+        <PrimaryToolbar { ...toolbarProps } />
+        <Table { ...tableProps }>
+            <TableHeader />
+            <TableBody />
+        </Table>
+        <TableToolbar isFooter>
+            <Pagination { ...toolbarProps.pagination } variant={ PaginationVariant.bottom } />
+        </TableToolbar>
+    </React.Fragment>;
+};
+
+export default ExampleTable;
```

Comment:
Observation: File could be renamed to `ExampleTable.js` to easily find it.

Reply:
I'll go for `TableToolTable.js`. It could also be used as a base for starting a custom table component and importing ExampleTable, might discourage that.